### ISSUE TITLE
Collect Madlib scenarios

### ISSUE THREAD

**Issue by mnichols08** [chingu-voyages/v43-tier1-team-09#23]
If you have any good ideas for a scenario. (We will probably want to write at least 5 good boilerplate paragraphs)

We can store them here centrally: https://docs.google.com/spreadsheets/d/1ZIR3LV7SLhk6dOJthe_HmtMfTCK9qnqKGFl0sSPs0D8/edit#gid=1122676451

**Comment by emilio12345** [comment]
I think there's plenty of scenarios in the spreadsheet for us to start voting on.

**Comment by Emimint** [comment]
Hey! I have actually made a separate xls here with more  scenarios: https://docs.google.com/spreadsheets/d/1EzkhbxhCYAiT6EwJdI8L7dH3jVlK428v/edit?usp=share_link&ouid=117065387916865563721&rtpof=true&sd=true . I guess at some point we can merge them in a single .csv or other format to work with.

**Comment by mnichols08** [comment]
I am sorry @Emimint that file is not accessible by the original spreadsheet so it will not allow me to do an `IMPORTRANGE()` and display it in our main file. I think it makes sense to have it in a separate file so we can export as a list.

I've created another spreadsheet within the Chingu project that we all have access to and titled it Scenarios, https://docs.google.com/spreadsheets/d/1fEvfBD2S4ZeGSy2Vv2eosXjYfP-Cokbi2BkMTIqwiCg/edit#gid=0 then imported all of the data from your file into it. 

Then I went in our main spreadsheet and ran an IMPORTRANGE as mentioned above on this new file in order to reflect any changes in both views at once.

With that said I think we are pretty solid on scenarios as mentioned so we can close this out to help finish up our sprint =)
